libass/ass_render.h

```diff
@@ -122,7 +123,8 @@ typedef struct {
 // GlyphInfo and TextInfo are used for text centering and word-wrapping operations
 typedef struct glyph_info {
     unsigned symbol;
-    unsigned skip;              // skip glyph when layouting text
+    bool skip;                  // skip glyph when layouting text
```

Comment:
To be honest, I’d say this was always meant to be a boolean and it should always have been an assignment rather than increment, but someone chose to do increment for shorter code or something. I left the increments unchanged because I generally prefer to not touch lines unnecessarily. Perhaps I’ll add a separate commit that just changes this to `bool` and changes the increments to assignments.

(FWIW, increment on booleans is apparently removed from C++17, but I see no sign of it being so much as deprecated in C.)